pkg/account/component.go

```diff
@@ -370,3 +387,41 @@ func (c *component) GetConfiguration(ctx context.Context, realmIDOverride string
 
 	return apiConfig, nil
 }
+
+func (c *component) SendVerifyEmail(ctx context.Context) error {
+	return c.executeActions(ctx, []string{ActionVerifyEmail})
+}
+
+func (c *component) SendVerifyPhoneNumber(ctx context.Context) error {
+	return c.executeActions(ctx, []string{ActionVerifyPhoneNumber})
+}
+
+func (c *component) executeActions(ctx context.Context, actions []string) error {
+	var accessToken = ctx.Value(cs.CtContextAccessToken).(string)
+	var currentRealm = ctx.Value(cs.CtContextRealm).(string)
+	var userID = ctx.Value(cs.CtContextUserID).(string)
+	var err = c.keycloakAccountClient.ExecuteActionsEmail(accessToken, currentRealm, actions)
+
+	if err != nil {
+		c.logger.Warn(ctx, "err", err.Error())
+		return err
+	}
+
+	if contains(actions, ActionVerifyEmail) {
+		var additionalInfos = map[string]string{"actions": strings.Join(actions, ",")}
+		var additionalBytes, _ = json.Marshal(additionalInfos)
+		var additionalString = string(additionalBytes)
+		c.reportEvent(ctx, "ACTION_EMAIL", database.CtEventRealmName, currentRealm, database.CtEventUserID, userID, database.CtEventAdditionalInfo, additionalString)
```

Comment:
ACTION_EMAIL must be sent only for VERIFY_EMAIL ? If yes, maybe we should have another specific event for verify phone number ?